docs/editing.md

```diff
@@ -41,6 +41,21 @@ At this point, the handling editor's job is to identify reviewers who have suffi
 
 To recruit reviewers, the handling editor can mention them in the `PRE-REVIEW` issue with their GitHub handle, ping them on Twitter, or email them. After expressing initial interest, candidate reviewers may need a longer explanation via email. See sample reviewer invitation email, below.
 
```

Comment:
```suggestion
**Reviewer Considerations**
- It is rare that all reviewers have the expertise to cover all aspects of a submission (e.g., knows the language really well and knows the scientific discipline well). As such, a good practice is to try and make sure that between the two or three reviewers, all aspects of the submission are covered. 
- Reviewers should conform to the [JOSS COI policy](https://joss.theoj.org/about#ethics). If a conflict exists, the procedure for addressing is outlined [here](https://joss.readthedocs.io/en/latest/reviewer_guidelines.html#joss-conflict-of-interest-policy). At least one reviewer with no conflict is required for each review. 
```

Note that the statement about one reviewer with no conflict is my understanding of the current policy based not on the docs but on editor emails. If I'm mistaken, please change/ignore.

Reply:
I would prefer to take out the last sentence, and might change the second suggested point to more simply:

- Selection and assignment of reviewers should adhere to the [JOSS COI policy](https://joss.readthedocs.io/en/latest/reviewer_guidelines.html#joss-conflict-of-interest-policy).

Reply:
Have edited based on your comments @danielskatz. Thanks!